### ISSUE TITLE
Problem with daloradius profile

### ISSUE THREAD

**Issue by mustafanass** [lirantal/daloradius#65]
Hi there 
I install daloradius in centoe7 machine but I get some problem with it 
When I created user without profile I get access accept but when I created profile with access-peroid check or max all session and add user to this profile I get access reject . 

I use maradb and php7 and httpd httpd-devel 
I test it with mikrotik and the internal freeradius with raddtest any help 
Thx

**Comment by lirantal** [comment]
Have you checked your FR logs?
Access-Period needs to be explicitly enabled using custom config in FR

**Comment by mustafanass** [comment]
yes i add config in main file radiusd.conf in modules section and this what i add 
sqlcounter accessperiod {
                   sql_module_instance = sql
                   dialect = ${modules.sql.dialect}
                   counter_name = Max-Access-Period-Never
                   check_name = Access-Period

                   key = User-Name
                   reset = never
                   query= "SELECT TIMESTAMPDIFF(SECOND, acctstarttime, NOW()) FROM radacct WHERE UserName='%{User-Name}' ORD$
    }

and import it from theirs name (accessperiod) in  Instantiation section i get access accept when i use radtest but when users get all limt in access-period its didnt disable and stay online and get access also i add  Acct-Interim-Interval and its same , and same things with max-all-session , but when i use session time out its work perfect and no problem with it . i work with centos 7 and php 7 

* i add this in /mods-available/counter 

sqlcounter accessperiod {
                   sql_module_instance = sql
                   dialect = ${modules.sql.dialect}
                   counter_name = Max-Access-Period-Never
                   check_name = Access-Period

                   key = User-Name
                   reset = never
                   query= "SELECT TIMESTAMPDIFF(SECOND, acctstarttime, NOW()) FROM radacct WHERE UserName='%{User-Name}' ORD$
    }

and its same its didnt work 
thx

**Comment by kkwised** [comment]
Hi,
@mustafanass .  You've done excellent so far.  A freeradius debug will help.  Also which freeradius version are you using? If version 3 you may need to add your counter to mod-enabled
